knotx-example/knotx-example-app/src/main/resources/logback.xml

```diff
@@ -38,6 +38,20 @@
     </encoder>
   </appender>
 
+  <appender name="access" class="ch.qos.logback.core.FileAppender">
```

Comment:
Maybe it is also worth to update `example` in the same way as `standalone` (logging to file appender instead of `STDOUT`)?
Or maybe in the example everything should be logged int `STDOUT`?